.github/workflows/build.yml

```diff
@@ -84,10 +85,10 @@ jobs:
       - name: Mycroft - install Mycroft
         run: |
           sudo apt update
-          git clone https://github.com/MycroftAI/mycroft-core.git /opt/mycroft-core
+          git clone --depth 1 --branch ${{ env.MYCROFT_CORE_BRANCH }} https://github.com/MycroftAI/mycroft-core.git /opt/mycroft-core
           ls /opt
           chmod +x /opt/mycroft-core/dev_setup.sh
-          /opt/mycroft-core/dev_setup.sh --allow-root -ci -sm
```

Comment:
should the `-ci` flag remain so the setup process doesn't prompt with questions?

Reply:
No, arg `-ci` was used in the past when I did some changes to `dev_setu.sh` to run in CI. In latest released version of it, setup look for `$CI`  environment variable, so questions are handled automatically.

ref: https://github.com/MycroftAI/mycroft-core/blob/5f964f7d4036210c7e1977e6903d3f13421e2af2/dev_setup.sh#L175

Reply:
Working workflow  can be seen there: [](https://github.com/Tony763/skill-homeassistant/actions/runs/3240230182/jobs/5310605787)